sdk/batch/Microsoft.Azure.Batch/Tools/CodeGenerationLibrary/CodeGenerationUtilities.cs

```diff
@@ -179,7 +187,7 @@ private static string GetProtocolCollectionToObjectModelCollectionString(string
                 return omPropertyData.GenericTypeParameter + ".ConvertFromProtocolCollectionReadOnly(" + protocolObjectSimpleGetter + ")";
             }
 
-            if (omPropertyData.HasPublicSetter)
+             if (omPropertyData.HasPublicSetter)
```

Comment:
Nit: Looks like an extra space snuck in here

Reply:
fixed